spec/support/images/topology_image.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100"><defs><style>.cls-1{fill:#d71e00}.cls-2{fill:#c21a00}.cls-3{fill:#eaeaea}.cls-4{fill:#fff}.cls-5{fill:#b7b7b7}</style></defs><title>Logo</title><g id="Layer_1" data-name="Layer 1"><circle class="cls-1" cx="50" cy="50" r="50" transform="rotate(-45 50 50)"/><path class="cls-2" d="M85.36 14.64a50 50 0 0 1-70.72 70.72z"/><path class="cls-3" d="M74.53 56.75C54.26 56 53.83 76.51 32.8 74.38 22 73.29 14 61.9 16.17 54.58l6-18.58c-.94 8.6 12.34 22.56 23.11 23.65 12.52 1.27 15.57-4.28 29.2-2.9"/><path class="cls-4" d="M80.47 39.21C69.47 38.1 61.79 49.34 54 59.42c6.35-1.06 10.67-3.66 20.5-2.67h.06c4.76.48 4.95 3.57 4.73 5l6.46-18c.25-1.46-.75-4.08-5.28-4.54z"/><path class="cls-5" d="M74.59 56.76h-.08a22.39 22.39 0 0 0-14.36 4.16 17.61 17.61 0 0 0 10.3 5c3.52.36 7.79-1.68 8.87-4.22.22-1.38.03-4.46-4.73-4.94zM68.5 43.06C60.68 36.85 57.07 25 43.2 23.61c-14.35-1.45-19.41 7.89-20.92 12.06l-.07.33c-.26 2.4.59 5.23 2.18 8.07 3.21-1.71 7.57-2.57 13.6-2 8.2.83 12.83 8.19 18.14 14.53C60.06 51.53 64 46.43 68.5 43.06z"/></g></svg>
```

Comment:
This support svg tag doesn't include the `XML_HEADER` is that an indication that ImageMagick is a one off in regards to that header?

Reply:
Yes that is correct. This is only necessary for topology's images it appears, like @Hyperkid123 pointed out it can be valid SVG without the XML header, but it isn't valid XML. Since ImageMagick uses libxml2 to parse SVG images the header is needed. Trying to find a way to work around this.

Reply:
Moved some logic to the Image class to catch if ImageMagick fails, to then try again with the XML header on it (just to generate the `extension` and `hash_code` fields), then it doesn't alter the SVG content. That way we aren't altering the responses at all, as well as handling if SVG data is passed in without the header.